omron_os32c_driver/src/os32c_node.cpp

```diff
@@ -113,6 +123,10 @@ int main(int argc, char *argv[])
       laserscan_msg.header.seq++;
       laserscan_pub.publish(laserscan_msg);
 
+      // Tick the frequency and update the diagnostics
+      frequency_status.tick();
+      updater.update();
```

Comment:
Any reason to do this separately vs. using DiagnosedPublisher?

http://docs.ros.org/api/diagnostic_updater/html/classdiagnostic__updater_1_1DiagnosedPublisher.html

Reply:
Thanks for the review, I was not aware of the `DiagnosedPublisher`